src/bin/observable.ts

```diff
@@ -123,14 +123,17 @@ try {
           "no-build": {
             type: "boolean",
             description: "Don’t build before deploying; deploy as is"
-          }
+          },
+          manifest: {type: "boolean", description: "Use the new upload strategy"},
+          "no-manifest": {type: "boolean", description: "Use the old upload strategy"}
```

Comment:
If these are for internal use only, and we plan to remove them before releasing, can you remove the descriptions or change them to “[internal use only]”? And add a “TODO remove me” comment? But I don’t really want to merge this to main if we don’t want to release this feature yet because 1.7 is imminent (and in general I prefer if we only merge to main what’s ready to release).

Reply:
I don't have any concrete reasons to think this isn't ready, I'm just being cautious. In the name of tempo, I removed the flag to disable the new feature, and we can fix it if there is a problem.